sympy/concrete/summations.py

```diff
@@ -520,9 +520,10 @@ def is_convergent(self):
                     pass
 
         ### ------------- Limit comparison test -----------###
-        # (n) comparison
-        lim_comp = limit((sequence_term/sym), sym, oo)
-        if lim_comp is not S.Infinity:
+        # (1/n) comparison
+        ref_sequence_term = Sum(1/sym, (sym, 1, S.Infinity)).function
```

Comment:
Is `ref_sequence_term == 1/sym`?

Reply:
Yes.

Reply:
There should be no need to create a Sum object for this expression.

Reply:
Updated @jksuom